minvws/nl-kat-coordination
# Add start time to scheduled reports

### Changes
This PR adds a start time to the scheduled reports. 
When scheduling a report, a user can now choose a start time as well. (Default is the current time, UTC)

### Issue link
Closes #3738

### Demo
![afbeelding](https://github.com/user-attachments/assets/d6b4827a-23c7-4417-8611-2954c6445a92)

![afbeelding](https://github.com/user-attachments/assets/9424ebcc-6d01-4fd2-910f-dcbe96eac6c9)


### QA notes
- Schedule a report
- Change the scheduled date and time
- Check if everything is as expected on the Scheduled Reports page

---

### Code Checklist







---

## Checklist for code reviewers:

Copy-paste the checklist from [the docs/source/templates folder](https://github.com/minvws/nl-kat-coordination/blob/main/docs/source/templates/pull_request_template_review_code.md) into your comment.

---

## Checklist for QA:

Copy-paste the checklist from [the docs/source/templates folder](https://github.com/minvws/nl-kat-coordination/blob/main/docs/source/templates/pull_request_template_review_qa.md) into your comment.
